core/geometry/src/test/Geometry.test.ts

```diff
@@ -0,0 +1,133 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { assert } from "chai";
+
+import { Cloneable, Geometry } from "../Geometry";
+
+describe("Transform.isAlmostEqualOptional", () => {
```

Comment:
These `describe` and `it` statements should be named with prefix "Geometry" not "Transform". :)
```suggestion
describe("Geometry.isAlmostEqualOptional", () => {
```

Reply:
copy paste issue!